feature-utils/poly-look/src/react-components/popUps/popUpBanner.css

```diff
@@ -0,0 +1,19 @@
+.pop-up-banner {
+    height: 56px;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    padding: calc(var(--gdu) * 4);
+    margin: calc(var(--gdu) * -6) calc(var(--gdu) * -4) calc(var(--gdu) * 6) calc(var(--gdu) * -4);
+    box-sizing: border-box;
+}
+
+.unsuccessful {
+    background-color: var(--color-red-200);
+    color: var(--color-red-900);
+}
+
+.successful {
+    background-color: var(--color-green-200);
```

Comment:
Please check these for color-blindness accessibility.

Reply:
I'm going to check this with the designers. I assume we would need more contrast between the background and the text, but no idea what the best solution is.